doc/autoscale/experiment/README.md

```diff
@@ -158,3 +172,22 @@ serving together.
         ```
     1. Gernerate Experiment Report
         After all the passes are finished, the report will generated at './out' folder.
+
+
+## Conclusions
+
+### Resource utilization increased in both cases
+
+Utilization increased by XX% in case one, and XX% in case two
+
+### Average Pending time reduced in both cases
+
+XX% in case one and XX% in case two.
+
+### Average execution time reduced in both cases
+
+XX% in case one and XX% in case two.
+
+### Improved the service quality with Hybrid cloud
```

Comment:
Hybrid don't need capitalization (hybrid), maybe "general purpose cluster" is easier to understand?

Reply:
corrected, thanks!